pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -88,6 +91,20 @@ type Logger struct {
 	Mode LoggerMode `json:"mode,omitempty"`
 }
 
+// Batcher provides optional payload batcher for all endpoints
+// +experimental
+type Batcher struct {
+	// MaxBatchSize of batcher service
+	// +optional
+	MaxBatchSize *string `json:"maxBatchSize,omitempty"`
```

Comment:
This should be an int32, right?

Reply:
This is described in https://github.com/kubeflow/kfserving/issues/890. It will be modified in the next version.